src/api/spec/controllers/webui/requests/change_devel_projects_controller_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe Webui::Requests::ChangeDevelProjectsController, type: :controller, vcr: true do
```

Comment:
Moving this Rspec controller would render a lot of vcr cassettes useless, right?

Reply:
I would say, [all of those](https://github.com/openSUSE/open-build-service/tree/master/src/api/spec/cassettes/Webui_RequestController/POST_changerequest)

Reply:
The cassettes you point out belong to the ['POST #changerequest'](https://github.com/openSUSE/open-build-service/blob/481c17c029b9da690957afde116e2606eee9aa58/src/api/spec/controllers/webui/request_controller_spec.rb#L268) section, still valid.